src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -235,6 +245,59 @@ public void runAppCfgCommand(List<String> args) throws ProcessRunnerException {
     processRunner.run(command.toArray(new String[command.size()]));
   }
 
+  /**
+   * Returns the version of the Cloud Sdk installation. Unlike other methods in this class that call
+   * gcloud, this method always uses a synchronous ProcessRunner and will block until the gcloud
+   * process returns.
+   *
+   * @throws ProcessRunnerException when process runner encounters an error
+   */
+  public String getVersion() throws ProcessRunnerException {
+    validateCloudSdk();
+
+    SynchronousOutputProcessRunner runner = new SynchronousOutputProcessRunner.Builder().build();
+
+    // gcloud info --format="value(basic.version)"
+    List<String> command = Lists.newArrayList(
+        getGCloudPath().toString(), "info");
+
+    command.addAll(GcloudArgs.get("format", "value(basic.version)"));
+
+    // TODO handle error exit codes
```

Comment:
Actually, I think this should be handled in the same way as the other commands, i.e., add exit hooks and let the client handle that any way they want. This is related to my higher level comment on the PR main thread.